.github/workflows/meilisearch-prototype-tests.yml

```
@@ -35,37 +35,35 @@ jobs:
-          - '7700:7700'
+          - "7700:7700"
     strategy:
       matrix:
-        node: ['20', '22']
+        node: ["20", "22"]
     name: integration-tests (Node.js ${{ matrix.node }})
     steps:
       - uses: actions/checkout@v6
       - name: Setup node
-        uses: actions/setup-node@v6
+        uses: ./.github/actions/set-up-node
```

actionlint (1.7.9)

description is required in metadata of "Set up Node.js and install dependencies" action at "/home/jailuser/git/.github/actions/set-up-node/action.yml"

(action)